docs/hoon/twig/cen-call/sig-open.md

```diff
@@ -13,6 +13,10 @@ sort: 3
 :rap(p :make(q +6 r))
 ```
 
+```
+=<(p %=(q +6 r))
+```
+
```

Comment:
And also deprecated :-)

But whatever.  Clearly we can't do these things in the usual "Curtis sits
in his basement and decides what to deprecate" way anymore...

On Thu, Jun 23, 2016 at 12:20 PM, Joe Bryan <notifications@github.com>
wrote:

> In docs/hoon/twig/cen-call/sig-open.md
> <https://github.com/urbit/docs/pull/55#discussion_r68298123>:
>
> > @@ -13,6 +13,10 @@ sort: 3
> >  :rap(p :make(q +6 r))
> >  ```
> >
> > +```
> > +=<(p %=(q +6 r))
> > +```
> > +
>
> Thanks, that's helpful context. And, ironically enough, %* is, in fact,
> undocumented ;)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/urbit/docs/pull/55/files/fb89b530694cb031b6e33d0b360f304e6cb77089#r68298123>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AALyAch1msgifTZMvBEtOclDXM0n_T99ks5qOtyWgaJpZM4I9D6c>
> .
>